Zend/zend_execute.c

```diff
@@ -2813,9 +2830,19 @@ static zend_always_inline void zend_fetch_property_address(zval *result, zval *c
 			ptr = OBJ_PROP(zobj, prop_offset);
 			if (EXPECTED(Z_TYPE_P(ptr) != IS_UNDEF)) {
 				ZVAL_INDIRECT(result, ptr);
-				if (flags) {
```

Comment:
Why did you remove this check? For some cases, it should be evaluated at compile-time.

Reply:
The flags check isn't completely removed, but moved into the condition (line 2845) because the other check always needs prop_info.